vcbuild.bat

```diff
@@ -13,9 +13,8 @@ if /i "%1"=="/?" goto help
 
 @rem Process arguments.
 set config=Release
-set msiplatform=x86
```

Comment:
It seems that before your changes, the msiplatform would be "x86" when the target_arch was "ia32". With your changes, it becomes redundant :+1: